dev/boot/src/platform.ts

```diff
@@ -41,6 +41,7 @@ const platform = createPlatform()
 
 platform.setMetadata(ui.metadata.LoginApplication, 'login')
 platform.setMetadata(ui.metadata.DefaultApplication, 'workbench')
+platform.setMetadata(ui.metadata.SettingApplication, 'setting')
```

Comment:
I guess `setting` page should be a part of `login` application. The reason is I can't envision `setting` app to be deployed separately without `login` one. 

@haiodo, I know we do not have routing docs yet, but I believe login <-> setting navigation should be done using some in-application routing mechanisms (and definitely possible using internal state for now).

So my proposal is not to expose `setting` as separate application.

Reply:
Working on docs right now, yes, 
it is possible to define a router inside login main page and have two variants like settings and login itself.